app/learningMessages.js

```diff
@@ -0,0 +1,51 @@
+define(['jquery', 'app/lang', 'app/timer', 'app/string'], function ($, lang, timer, string) {
+  /* Function show for displaying normal, danger and warning messages.
+  * The message is included as html and a class with the corresponding type is
+  * added. Within the switch statement the icon is added.
+  */
+
+  return {
+    show: function(message, type, countdown) {
+      if (countdown > 0) {
+        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">" + lang("message_enterorwait", "<span class=\"countdown\" data-seconds=\"" + countdown + "\"></span>"));
```

Comment:
I remember that @sdejong was working on this one for a while and couldn't figure it out. What was it again Sander?